docs/en/change-backend-provider.md

```diff
@@ -0,0 +1,14 @@
+# Change backend provider
+
+The default backend provider uses session based storage which stores the cart into a serialized
+in-memory array. This can be changed to any class which implements the `DMSCartBackendInterface`
+interface using the Injector:
+
+```yaml
+Injector:
+    DMSDocumentCart:
+      properties:
+        backend: %$NewBackendService
+```
+
+In the above example, `%$NewBackendService` should be a valid class which implements `DMSCartBackendInterface` interface.
```

Comment:
I'm not sure if this matches the implied effects of the constructor of this class. Perhaps we need to remove the constructor in favour of injector dependencies instead. Also re "should be a class that implements DMSCartBackendInterface" - sure, should we enforce that?